src/bootslide.js

```diff
@@ -249,16 +249,19 @@ Bootslide.prototype.buildSections = function (menu, sections, backTo) {
       if (target.contentEndpoint) {
         var newSection = addAndContinue()
 
-        a.click(function (e) {
+        a.on('click', function (e) {
           prepareContentEndpoint(newSection)
         })
-      } else if (typeof target.target === 'function' || (typeof target.target === 'undefined' && self.defaultTarget)) {
+      // If this target is of layout 'tiles', let it's tiles handle the click event.
```

Comment:
Does that mean top-level tiles won't automatically go to the submenu anymore? Or is that case handled earlier already?

Reply:
Top level tiles are advancing to their targets regardless of layout after this change.
I'm not 100% sure that's what you're asking so here's what it looks like:

![2021-06-02 10 09 26](https://user-images.githubusercontent.com/7981518/120495519-c7356300-c38a-11eb-87f0-4b262dacd8ed.gif)

Reply:
Yup, that's what I was asking, thanks!